src/QmlControls/ParameterEditorDialog.qml

```diff
@@ -138,6 +149,26 @@ QGCViewDialog {
                 }
             }
 
+            Column {
+                Repeater {
+                    id: bitmaskEditor
+                    model: fact.bitmaskStrings
+                    delegate : QGCCheckBox {
+                        text : modelData
+                        checked : fact.value & fact.bitmaskValues[index]
+                        function check_uncheck_all() {
+                            if (text.toLowerCase() == "all") {
```

Comment:
I don't think there is going to be a way to do this. There isn't enough informatio in the meta data.